### PR TITLE
removed unused classes

### PR DESCRIPTION
**Pull request by lgreenlaw2022**

### CODE REVIEW

**Review by davewood-maine — DISMISSED** [review]

**Review by snutt — DISMISSED** [review]

**Comment by rpbritton** [comment]
oops

File: DeepSpace/src/main/cpp/Robot.cpp [hunk 1]
```diff
@@ -24,9 +23,9 @@ void Robot::RobotInit() {
   // Initialize Subsystems
   m_driveTrain.reset(new BrushlessDrive());
 
-  m_chooser.SetDefaultOption("Default Auto", &m_defaultAuto);
```
Comment (rpbritton) [inline DeepSpace/src/main/cpp/Robot.cpp:27]: These lines are commented while the other lines below are deleted.

File: DeepSpace/src/main/cpp/Robot.cpp [hunk 2]
```diff
@@ -72,11 +71,6 @@ void Robot::AutonomousInit() {
   //   m_autonomousCommand = &m_defaultAuto;
   // }
 
-  m_autonomousCommand = m_chooser.GetSelected();
```
Comment (rpbritton) [inline DeepSpace/src/main/cpp/Robot.cpp:75]: These are deleted, while others are commented.

**Review by rpbritton — COMMENTED** [review]
Good, just wondering if all `m_autonomousCommand` stuff should be removed or deleted.

**Comment by rpbritton** [comment]
Commented out example commands should be removed now or else they probably won't be later.

**Review by sphillip2021 — APPROVED** [review]
Good stuff!

**Comment by rpbritton** [comment]
> 
> 
> Good stuff!

Thanks Scott!

**Pull request merged**